### PR TITLE
LP144 make divider element

### PR DESCRIPTION
**Pull request by Usiel-Gandalf**
Divider element maked

In this element, an divider element was created with the width and color of leaspert requeriments

https://devaptivist.atlassian.net/browse/LP-144?atlOrigin=eyJpIjoiNGQzODg0MDNlNTNhNDBiMTljZGJmMWMyN2E3NjBkNjkiLCJwIjoiaiJ9

divider element

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.iteneum.designsystem.components
+
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.designsystem.theme.LeasePertTheme
+
+@Composable
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt:17]: Can you add the information of this component? please

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.iteneum.designsystem.components
+
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.designsystem.theme.LeasePertTheme
+
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt:9]: Please, add the documentation to this function.

Comment (Usiel-Gandalf) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt:9]: documentation added

**Review by melissaquezada94 — APPROVED** [review]
You're doing well!

**Review by carlosph816 — APPROVED** [review]
Well done!! FF

**Pull request merged**